### PR TITLE
Add UnturnedChat.Say overloads with iconURL parameter

### PR DESCRIPTION
**Pull request by Tortellio**

### CODE REVIEW

File: Rocket.Unturned/Chat/UnturnedChat.cs [hunk 1]
```diff
@@ -116,11 +131,28 @@ public static void Say(IRocketPlayer player, string message, Color color, bool r
...
 
+        public static void Say(IRocketPlayer player, string message, Color color, string iconURL, bool rich)
+        {
+            if (player is ConsolePlayer)
+            {
+                Core.Logging.Logger.Log(message, ConsoleColor.Gray);
+            }
+            else
+            {
+                Say(new CSteamID(ulong.Parse(player.Id)), message, color, iconURL);
```
Comment (CyberAndrii) [inline Rocket.Unturned/Chat/UnturnedChat.cs:142]: The `rich` parameter is defined but not used, so rich text will not work. Same issue as I fixed in #3.

Comment (Tortellio) [reply Rocket.Unturned/Chat/UnturnedChat.cs]: oh thanks didn't see that, fixed

**Comment by SDGNelson** [comment]
Why use `UnturnedChat.Say` rather than `ChatManager.serverSendMessage`? The official method will remain backwards compatible and supports all features.

**Comment by Trojaner** [comment]
New plugins should use ChatManager.serverSendMessage, but this bug should be fixed

**Comment by SDGNelson** [comment]
The `rich` parameter bug mentioned will be fixed in the next release!

**Comment by Tortellio** [comment]
thank you

**Pull request closed without merging**